library/core/src/main/java/com/google/android/exoplayer2/text/cea/Cea708Decoder.java

```diff
@@ -270,10 +278,11 @@ private void finalizeCurrentPacket() {
   @RequiresNonNull("currentDtvCcPacket")
   private void processCurrentPacket() {
     if (currentDtvCcPacket.currentIndex != (currentDtvCcPacket.packetSize * 2 - 1)) {
-      Log.w(TAG, "DtvCcPacket ended prematurely; size is " + (currentDtvCcPacket.packetSize * 2 - 1)
+      Log.d(TAG, "DtvCcPacket ended prematurely; size is " + (currentDtvCcPacket.packetSize * 2 - 1)
           + ", but current index is " + currentDtvCcPacket.currentIndex + " (sequence number "
-          + currentDtvCcPacket.sequenceNumber + "); ignoring packet");
-      return;
+          + currentDtvCcPacket.sequenceNumber + ");");
+      // This is not invalid packet. As per CEA-708 section 4.4.1.1, the Packect end can happen
+      // when the reception of cc_type = 0x03 (binary 11).
```

Comment:
I'm going to update this comment to the following - does that sound right to you?

```
// We've received cc_type=0x03 (packet start) before receiving packetSize byte pairs of data.
// This might indicate a byte pair has been lost, but we'll still attempt to process the data
// we have received.
```

Reply:
@icbaker I agree with the above comment.